app/src/main/res/values-night/colors.xml

```diff
@@ -77,6 +77,7 @@
     <color name="top_site_title_text">@color/top_site_title_text_dark_theme</color>
     <color name="top_site_pager_dot">#3A3944</color>
     <color name="top_site_pager_dot_selected">#5B5B66</color>
+    <color name="top_site_pin_icon_normal_theme">@color/top_site_pin_icon_dark_theme</color>
```

Comment:
You can specify the dark theme color theme. Let's also rename this to `top_site_pin_icon_background_tint`

```suggestion
    <color name="top_site_pin_icon_background_tint">@color/violet_50</color>
```